indico/modules/events/editing/client/js/editing/timeline/ReviewForm.jsx

```diff
@@ -171,7 +171,7 @@ export default function ReviewForm() {
           {!judgmentType && judgmentForm}
           <FinalForm
             initialValues={{
-              comment: '',
+              comment: textAreaValue,
```

Comment:
This breaks "request changes" when you've first written the comment (as a comment) and then switched to judging instead: Now you have to make some change to the textarea before you can submit the judgment.